packages/eslint-plugin-lit-a11y/lib/rules/accessible-emoji.js

```diff
@@ -40,19 +46,15 @@ module.exports = {
 
     return {
       TaggedTemplateExpression: node => {
-        if (
-          node.type === 'TaggedTemplateExpression' &&
-          node.tag.type === 'Identifier' &&
-          node.tag.name === 'html'
-        ) {
+        if (isHtmlTaggedTemplate(node)) {
           const analyzer = TemplateAnalyzer.create(node);
 
           analyzer.traverse({
             enterElement(element) {
               // todo: figure out what is meant with literalChildValue, try to see from tests
               const literalChildValue = element.children.find(child => child.type === 'text');
 
-              if (literalChildValue && emojiRegex().test(literalChildValue.data)) {
+              if (literalChildValue && EMOJI_REGEXP.test(literalChildValue.data)) {
```

Comment:
does this need to get an instance at runtime, or is once at module init enough?

Reply:
at module init should be good enough yeah